MediaBrowser.sln

```diff
@@ -72,7 +72,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Jellyfin.Networking.Tests",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Jellyfin.Dlna.Tests", "tests\Jellyfin.Dlna.Tests\Jellyfin.Dlna.Tests.csproj", "{B8AE4B9D-E8D3-4B03-A95E-7FD8CECECC50}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Jellyfin.XbmcMetadata.Tests", "tests\Jellyfin.XbmcMetadata.Tests\Jellyfin.XbmcMetadata.Tests.csproj", "{30922383-D513-4F4D-B890-A940B57FA353}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Jellyfin.XbmcMetadata.Tests", "tests\Jellyfin.XbmcMetadata.Tests\Jellyfin.XbmcMetadata.Tests.csproj", "{30922383-D513-4F4D-B890-A940B57FA353}"
```

Comment:
Why did this project guid change?

Reply:
First I'll naturaly update the PR to restore the previous guid, but now the analisys that is quite a rabbithole.

So all the projects already use "correctly" the guid {9A19103F-16F7-4668-BE54-9A1E7A4F7556} (from now on "9A") as the project type guid
the exception being Jellyfin.XbmcMetadata.Tests that uses {FAE04EC0-301F-11D3-BF4B-00C04F79EFBC} (from now on "FA") due to being the last project added (see below)
 
I put correctly between quotes because as commented by a member of the Dot plaftorm in the solution there should be only the legacy guid "FA" https://github.com/dotnet/project-system/issues/1821#issuecomment-288239298
for compatibility reason but apparently VS like to use both:
New projects are added with GUID "FA" but the next time the solution file is updated, the existing ones are changed to "9A".

Bonus point: VS for Mac does the opposite

https://github.com/dotnet/project-system/issues/1821